### PR TITLE
fix: homepage content and design

### LINKED ISSUE #26: Homepage content and design
- [x] stats on contributions
- [x] hero image from Israel site
- [x] add stats icons

### LINKED ISSUE #54: google analytics
add google analytics to better track page usage

### PR DESCRIPTION
**Pull request by bennypowers**
closes #54

### CODE REVIEW

**Comment by TomerFi** [comment]
This needs rebasing after switching to new repo.

File: assets/css/index.css [hunk 1]
```diff
@@ -69,6 +69,33 @@ rh-footer {
   }
 }
 
+#home{
```
Comment (bennypowers) [inline assets/css/index.css:76]: i'm planning to make some changes. mostly code formatting but also some fixes so the HTML

**Comment by bennypowers** [comment]
When https://github.com/RedHat-UX/red-hat-design-system/pull/652 releases, we'll be able to upgrade and remove the patch file

File: assets/css/index.css [hunk 2]
```diff
@@ -195,7 +197,7 @@ rh-global-footer {
         }
         &.active {
           & a {
-            background: var(--rh-color-surface-lighter);
+            background: gray;
```
Comment (bennypowers) [inline assets/css/index.css]: These should be reverted, we should use the token values for brand consistency

Comment (ilan-pinto) [reply assets/css/index.css]: fixed

**Review by ilan-pinto — APPROVED** [review]

**Pull request merged**